easybuild/easyconfigs/a/AOMP/AOMP-13.0-2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors:: Jorgen Nordmoen <jorgen.nordmoen@usit.uio.no>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'AOMP'
+version = '13.0-2'
+
+homepage = 'https://github.com/ROCm-Developer-Tools/aomp'
+description = "AMD fork of LLVM, setup for OpenMP offloading to Accelerators"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/ROCm-Developer-Tools/aomp/releases/download/rel_%(version)s/']
+sources = ['aomp-%(version)s.tar.gz']
+patches = ['build_aomp-%(version)s.patch']
```

Comment:
Please rename the patch file to something like `AOMP-13.0.2_short-description.patch`

Reply:
Fixed